plugins/importexport/native/filter/filterConfig.xml

```diff
@@ -109,7 +109,7 @@
 			displayName="plugins.importexport.native.displayName"
 			description="plugins.importexport.native.description"
 			inputType="xml::schema(plugins/importexport/native/native.xsd)"
-			outputType="class::classes.publication.Publication" />
+			outputType="class::classes.publication.Publication[]" />
```

Comment:
I think this also requires a database migration because it's written to the `filter_groups` table (unless there's already a migration and this is just clean-up).